hp/src/CodeGenerator.hs

```diff
@@ -265,10 +265,11 @@ generateCondition (Condition ifCond thenStat elseStat) =
     ++ ")"
 
 generateCycle :: Cycle -> String
-generateCycle cy = ""
+generateCycle (Cycle mapF mapL) = "map(" ++ generateIdentifier mapF ++ ", " ++ generateCycleList mapL ++ ")"
```

Comment:
Please fix the order, I mean, you are generating this:
![Screenshot from 2024-07-04 19-27-07](https://github.com/luiggymamani/hufflepuff-own-language/assets/135136456/873453c5-b650-40b2-9758-81f766f80b26)

but this is expected:
![Screenshot from 2024-07-04 19-29-09](https://github.com/luiggymamani/hufflepuff-own-language/assets/135136456/9edf8095-ab02-497a-ae01-6deeb0167a75)

Reply:
observations solved